### PR TITLE
Update Alight-observability.yaml

### PR DESCRIPTION
**Pull request by LokeshChukka**

### CODE REVIEW

File: workflow-templates/Alight-observability.yaml [hunk 1]
```diff
@@ -4,14 +4,30 @@ on:
...
       - name: generate-html
-        uses: AlightEngineering/devops-observability@devops-676
+        uses: AlightEngineering/devops-observability@main
         with:
           api-url: ${{ inputs.url }}
+          styles-file: ${{ env.json_path }}
+          
+     
+
+
```
Comment (vijendermali) [inline workflow-templates/Alight-observability.yaml:33]: Remove these Additional Lines

File: workflow-templates/Alight-observability.yaml [hunk 2]
```diff
@@ -4,14 +4,30 @@ on:
...
     steps:
+      - name: set default json path
+        if: ${{ inputs.style == 'styles.json' }}
+        run: echo "json_path=${{ inputs.style }}"  >> $GITHUB_ENV
+      - name: set user path
+        if: ${{ inputs.style != 'styles.json' }}
+        run: echo "json_path=${{ github.workspace }}/${{ inputs.style }}"  >> $GITHUB_ENV 
       - name: generate-html
-        uses: AlightEngineering/devops-observability@devops-676
+        uses: AlightEngineering/devops-observability@main
```
Comment (vijendermali) [inline workflow-templates/Alight-observability.yaml:26]: Is the Code merged to **main** branch in **devops-observability** repo from **devops-676** branch?

File: workflow-templates/Alight-observability.yaml (hunk 1 above)
Comment (LokeshChukka) [reply workflow-templates/Alight-observability.yaml]: yes, Removed those unwanted lines.

File: workflow-templates/Alight-observability.yaml (hunk 2 above)
Comment (LokeshChukka) [reply workflow-templates/Alight-observability.yaml:26]: yes. Merged from devops-676 to main branch.

**Pull request closed without merging**